### PR TITLE
feat(jump-links) 2.0 Add new ComposedEvent to jump-links

### PR DESCRIPTION
**Pull request by heyMP**
Updated jump-links based on your suggestions in the previous PR.  Thanks Benny 👍

### CODE REVIEW

File: elements/pfe-jump-links/pfe-jump-links-nav.ts [hunk 1]
```diff
@@ -33,16 +34,31 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
...
+  ) {
+    super('stuck');
+  }
+}
+
+export class ActiveItemEvent extends ComposedEvent {
+  constructor(
+    index: keyof Array,
+  ) {
+    super('active-item');
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.ts:49]: WDYT about this name? I was thinking `change` might be better, but we already use `change` in panel...

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: elements/pfe-jump-links/pfe-jump-links-nav.ts (hunk 1 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.ts:49]: maybe:
`active-item` => `change`
`change` => `content-change`

I think that whatever name we decide for panel contents changing we'll want to standardize for other elements reacting to mutation observer changes.

Comment (bennypowers) [reply elements/pfe-jump-links/pfe-jump-links-nav.ts:49]: let's defer that discussion to later, k?

File: elements/pfe-jump-links/pfe-jump-links-nav.ts [hunk 2]
```diff
@@ -33,16 +34,31 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
...
+  constructor(
+    public stuck: Boolean,
+  ) {
+    super('stuck');
+  }
+}
+
+export class ActiveItemEvent extends ComposedEvent {
+  constructor(
+    index: number,
```
Comment (bennypowers) [inline elements/pfe-jump-links/pfe-jump-links-nav.ts:47]:
```suggestion
    public index: number,
```

File: elements/pfe-jump-links/pfe-jump-links-nav.ts (hunk 1 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.ts]: Sounds good.  I made the changes to the event names and updated the tests to cover the new events.

File: elements/pfe-jump-links/pfe-jump-links-nav.ts (hunk 2 above)
Comment (heyMP) [reply elements/pfe-jump-links/pfe-jump-links-nav.ts]: Nice. I updated it in the latest commit.

**Pull request merged**